pennylane/qnn/torch.py

```diff
@@ -216,14 +212,10 @@ def __init__(self, qnode, weight_shapes: dict, init_method: Optional[Callable] =
         }
 
         # validate the QNode signature, and convert to a Torch QNode.
-        if qml.tape_mode_active():
-            # TODO: update the docstring regarding changes to restrictions when tape mode is default.
-            self._signature_validation_tape_mode(qnode, weight_shapes)
-            self.qnode = qnode
-            self.qnode.to_torch()
-        else:
-            self._signature_validation(qnode, weight_shapes)
-            self.qnode = to_torch(qnode)
+        # TODO: update the docstring regarding changes to restrictions when tape mode is default.
```

Comment:
Is this a todo that should be implemented here?

Reply:
Yes, was hoping @trbromley would be able to have a look since I think he wrote this TODO :)